.github/workflows/test-suite-desktop-e2e.yml

```
@@ -36,20 +36,38 @@ jobs:
+          #   CONTAINERS: "trezor-user-env-unix"
+          # - TEST_GROUP: "@group=wallet"
+          #   CONTAINERS: "trezor-user-env-unix bitcoin-regtest"
+          - TEST_GROUP: "@group=empty group produces error"
             CONTAINERS: "trezor-user-env-unix"
-          - TEST_GROUP: "@group=wallet"
-            CONTAINERS: "trezor-user-env-unix bitcoin-regtest"
-
     steps:
+      - name: Check if this job should be skipped
+        id: check_skip
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)